### DISCUSSION TITLE
Dealing with a minimp3 dependency

### DISCUSSION THREAD

**Discussion by SiegeLord** [Homebrew/discussions#2806] [Writing Formulae/Casks]
https://github.com/lieff/minimp3 is a convenient, relatively popular (based on the star number) header-only C library for decoding MP3 files. I want the library I help maintain ([Allegro](https://github.com/Homebrew/homebrew-core/tree/master/Formula/allegro)) to use it on when packaged via homebrew. Problem is, I think it runs afoul the acceptable formula requirements as it has no webpage and no releases. What would be the best way to proceed here?

- Should we (Allegro developers) vendor this library? (this too is discouraged, as I understand it)
- Maybe an exception can be made for this library?

Thanks for any advice.

**Comment by SMillerDev** [comment]
The main Github page is good enough as a webpage but I would really urge upstream to make releases so it can be packaged.
